docs/02-hugo.Rmd

```diff
@@ -1,28 +1,89 @@
 # Hugo
 
-What is a static site generator?
+In this chapter, we will introduce [Hugo](https://gohugo.io), the static site generator on which `blogdown` is based. 
 
-What is Hugo?
+"Static site generator" refers to the way that a website's content is generated. On dynamically generated sites, the content is rendered every time a user visits the site. Alternatively, on statically generated sites, the content is only rendered when the content changes, making it ready to access when a user visits the site^[More information here: https://gohugo.io/overview/introduction/]. 
 
-Why Hugo?
+Hugo is one example of a static site generator. It provides the framework for a variety of website types including, but certainly not limited to, blogs.
+
+We're fans of Hugo because it's
 
 - fast
 - easy to install (single binary)
 - a big community (Github)
 
 Although we think Hugo is a fantastic static site generator, there is really one and only one important but missing feature: the support for R Markdown. That is basically the whole point of the **blogdown** package. This missing feature means that you cannot easily generate results using R code on your web pages, since you can only use static Markdown documents. Besides, you have to use Hugo's choice of the Markdown engine named "Blackfriday" instead of the more powerful Pandoc.^[There is a feature request in Hugo's repository that has been three years old: https://github.com/spf13/hugo/issues/234, and it seems that it will not be implemented in the near future.]
 
-What does a typical file and directory structure look like?
+
+Hugo uses a consistent file and folder structure to create your website.  The rest of this chapter will give more detail on the following files and folders:
 
 - config.toml
 - content
 - static
 - themes
 - layouts
 
+```{r lithium, fig.cap='Files and Folders created when you create a new_site() using blogdown', echo=FALSE, fig.align='center', out.width='90%'}
+knitr::include_graphics('images/folder-structure.png')
+```
```

Comment:
1. (Nitpicking) Could you cut a little bit off the left margin of the screenshot? It has a border on the left but not on the right.

2. The figure caption will have to be written in a separate paragraph using text references because it contains an underscore (LaTeX special character): https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html#text-references